include/odp/api/spec/classification.h

```diff
@@ -105,6 +105,10 @@ typedef union odp_cls_pmr_terms_t {
 		uint64_t        custom_l3:1;
 		/** IGMP Group address */
 		uint64_t	igmp_grp_addr:1;
+		/** ICMP identifier */
+		uint64_t	icmp_id:1;
```

Comment:
I wonder if this is useful or just API bloat, i.e. are there use cases where it is necessary to be able to separate ICMP messages of the same type based on the identification value already in the classifier and not later in the SW? I also wonder why the identification field is important enough to get added here but other fields, like ICMP code, are not?

One can also use custom PMR terms to match things that are not explicitly supported, so maybe every possible protocol and field does not need to be added.

Reply:
I think, we can add new terms based on the need.